src/pages/questions/QuestionDisplay.jsx

```diff
@@ -57,16 +60,55 @@ const getValueTagExamplesURL = (question) => {
   return "";
 };
 
-const QuestionDisplay = ({ question, answerQuestion, resetFilters }) => {
+const getNbOfQuestionForValue = async ({ type, valueTag }) => {
+  const { data: dataFetched } = await robotoff.questions(
+    "random",
+    type,
+    valueTag,
+    null,
+    null,
+    1
+  );
+  return dataFetched.count;
+};
+
+const QuestionDisplay = ({
+  question,
+  answerQuestion,
+  resetFilters,
+  filterState,
+}) => {
   const { t } = useTranslation();
   const valueTagQuestionsURL = getValueTagQuestionsURL(question);
   const valueTagExamplesURL = getValueTagExamplesURL(question);
+  const [nbOfPotentialQuestion, setNbOfPotentialQuestions] =
+    React.useState(null);
 
   React.useEffect(() => {
+    if (filterState.valueTag) {
+      // value is already in the filter so it's a useless information
+      setNbOfPotentialQuestions(null);
+      return;
+    } else {
```

Comment:
no need of the else as there is a return, removing the else also reduces code nesting so easier to read